package.json

```diff
@@ -110,11 +110,12 @@
         "@types/chai": "^4.2.12",
         "@types/mocha": "^7.0.2",
         "@types/node": "^12.12.0",
```

Comment:
The verion should be bumped in this file too

Reply:
done in upcoming review fixes commit
I increased the version in the server and client to 1.0.0

Reply:
I meant the plugin's overall version. It is still 2.2.3, but should be 2.3.0